SofaKernel/modules/SofaCore/src/sofa/core/behavior/BaseMass.h

```diff
@@ -116,7 +116,7 @@ class SOFA_CORE_API BaseMass : public virtual objectmodel::BaseObject
     /// Get the matrix relative to the DOF at \a index.
     virtual void getElementMass(sofa::Index index, defaulttype::BaseMatrix *m) const = 0;
 
-    virtual bool isDiagonal() { return false; }
+    virtual bool isDiagonal() const { return false; }
```

Comment:
```suggestion
    virtual bool isDiagonal() const = 0;
```

Reply:
It forces HexahedralFEMForceFieldAndMass to implement the method. If I add the method, do I return false? fyi @hugtalbot